src/SME.SGP.Dominio.Servicos/ServicoNotificacaoFrequencia.cs

```diff
@@ -145,40 +152,49 @@ public async Task NotificarCompensacaoAusencia(long compensacaoId)
 
             repositorioNotificacaoCompensacaoAusencia.Excluir(compensacaoId);
 
-            var gestores = BuscaGestoresUe(ue.CodigoUe);
-            if (gestores != null && gestores.Any())
+            var cargos = new Cargo[] { Cargo.CP };
+            if (GerarNotificacaoExtemporanea(possuirPeriodoAberto, parametroAtivo != null ? bool.Parse(parametroAtivo.Valor) : false))
             {
-                foreach (var gestor in gestores)
-                {
-                    var notificacaoId = NotificarCompensacaoAusencia(compensacaoId
-                            , gestor.Usuario
-                            , professor.Nome
-                            , professor.CodigoRf
-                            , disciplinaEOL
-                            , turma.CodigoTurma
-                            , turma.Nome
-                            , turma.ModalidadeCodigo.ObterAtributo<DisplayAttribute>().ShortName
-                            , ue.CodigoUe
-                            , ue.Nome
-                            , ue.TipoEscola.ObterAtributo<DisplayAttribute>().ShortName
-                            , dre.CodigoDre
-                            , dre.Nome
-                            , compensacao.Bimestre
-                            , compensacao.Nome
-                            , alunosDto);
-
-                    // Grava vinculo de notificação x compensação
-                    repositorioNotificacaoCompensacaoAusencia.Inserir(notificacaoId, compensacaoId);
-                }
 
-                // Marca aluno como notificado
-                alunosDto.ForEach(alunoDto =>
-                {
-                    var aluno = alunos.FirstOrDefault(a => a.CodigoAluno == alunoDto.CodigoAluno);
-                    aluno.Notificado = true;
-                    repositorioCompensacaoAusenciaAluno.Salvar(aluno);
-                });
+                await NotificarCompensacaoExtemporanea(
+                     professor.Nome,
+                     professor.CodigoRf,
+                     disciplinaEOL,
+                     turma.CodigoTurma,
+                     turma.Nome,
+                     turma.ModalidadeCodigo.ObterAtributo<DisplayAttribute>().ShortName,
```

Comment:
Existe uma extensão pra isso
```suggestion
                     turma.ModalidadeCodigo.ObterNomeCurto(),
```